ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -114,7 +114,7 @@
         @Html.DisplayFor(model => model.Team.RoomID)
     </dd>
     -->
-                <!--
+                
                 <dt>
                     Team room
                 </dt>
```

Comment:
Line 130 doesn't match the create/edit pages -- it asks for a room, but the other pages wouldn't allow it

Reply:
It's allowed on create/edit if it's not a intern event (that's what the if/else is for) or if it's on the admin view for the intern event.